region/wreckedship/main/Bowling Alley.json

```
@@ -893,6 +893,30 @@
       "flashSuitChecked": true,
       "note": ["This requires a first frame Spikesuit unmorph."]
     },
+    {
+      "link": [2, 2],
+      "name": "X-Mode Shinecharge, R-Mode Pause Abuse Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "f_DefeatedPhantoon",
```

```suggestion
        "f_DefeatedPhantoon",
        "canBeVeryLucky",
```
Perhaps a notable fits here.  Maybe a new tech, `canRiskyRMode`.